RubeesEat/TagHelpers/WebTestActionTagHelper.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace RubeesEat.TagHelpers;
+
+[HtmlTargetElement(Attributes = "t-action")]
+public class WebTestActionTagHelper : TagHelper
+{ 
+    
+    private readonly IWebHostEnvironment _environment;
+    [HtmlAttributeName("t-action")] public string Action { get; set; }
+    [HtmlAttributeName("t-click-behavior")] public string ClickBehavior { get; set; }
+
+    public WebTestActionTagHelper(IWebHostEnvironment environment)
+    {
+        _environment = environment;
+    }
+    
+    public override void Process(TagHelperContext context, TagHelperOutput output)
+    {
+        if (!(_environment.IsProduction()))
+        {
+            output.Attributes.Add("data-test-action", Action);
+            
+            if (ClickBehavior == null)
+                ClickBehavior = "ClickAndWaitUntilStale";
```

Comment:
We can assign the property above to the default value. This way, we don't need to check for `null` here and can simplify the logic.